### ISSUE TITLE
Gjengfunksjon

### ISSUE THREAD

**Issue by Cobrazking** [LarsFredrikHolstTry/bawser#26]
Kom gjerne med noen innspill på hva du har sett for deg her.

Legger ved en layout av et eksempel på hvordan det kan være.

![Famile](https://user-images.githubusercontent.com/120334498/210989224-a9e6aecc-461a-410b-a40d-f60a2396a026.png)

**Comment by Cobrazking** [comment]
Tanken er at det er enkeltspillere som kan eie firmaer, men om enkeltpersonen blir med i en gjeng så er det gjengen som får profitten av firmaet. Dersom personen ikke er med i en gjeng så står personen fritt til å ta ut overskuddet selv, men det betyr også at han ikke er "beskyttet" av en gjeng som kan ta tilbake firmaet. 

Gjengranken baserer seg på antall kronasjer som er tjent inn totalt + EXP ranket totalt + Antall firmaer som eies. (si f.eks at det er en cap på 5 firmaer per gjeng).

**Comment by Cobrazking** [comment]
Hvis dette ser og høres greit ut kan jeg gå igang med undersider her :)

**Comment by LarsFredrikHolstTry** [comment]
Dette ser veldig bra ut! Men hvordan vil 5 firma cap'en funke dersom det er 6 medlemmer i en gjeng og alle har 1 firma hver?

**Comment by Cobrazking** [comment]
Tenker at da får ikke spilleren blitt med i gjengen før en av de andre eller hen selv har kvittet seg med firma.

**Comment by LarsFredrikHolstTry** [comment]
Tenker kanskje heller det gir mening å ha en cap på firma per bruker. Ettersom at gjeng blir mindre ettertraktet om man ikke tjener 100% av profitten på firmaet selv.

**Comment by Cobrazking** [comment]
Ja, kan være en god ide det. Kanskje det bør være mulig å bestemme om firmainntektene skal gå til gjengen eller brukeren inne på firmasiden? :)

**Comment by LarsFredrikHolstTry** [comment]
3 firmaer per bruker høres bra ut! Skulle vi laget et issue over hvilke firmaer vi skal ha også?

**Comment by Cobrazking** [comment]
Gjør det :)
